### PR TITLE
relval matrix updates 

### PR DESCRIPTION
**Pull request by hengne**
relval updates:
- add phase1 ZMM_13 pythia8 workflow
- update gensim input for phase1 
- finer job splitting for phase1 digi and reco steps
- update mixing/premixing inputs for 2015 workflows
- JpsiMuMu_Pt-15 workflow update to JpsiMuMu_Pt-8 with Pythia8

### CODE REVIEW

**Comment by hengne** [comment]
please test

File: Configuration/PyReleaseValidation/python/MatrixInjector.py [hunk 1]
```diff
@@ -192,6 +192,10 @@ def prepare(self,mReader, directories, mode='init'):
             wmsplit['RECOUP15']=5
             wmsplit['RECOAODUP15']=5
             wmsplit['DBLMINIAODMCUP15NODQM']=5
+            wmsplit['DigiFull']=5
+            wmsplit['RecoFull']=5
+            wmsplit['DigiFullPU']=1
+            wmsplit['RecoFullPU']=1
```
Comment (davidlange6) [inline Configuration/PyReleaseValidation/python/MatrixInjector.py]: these really effect only the very high pileup workflows?

Comment (hengne) [inline Configuration/PyReleaseValidation/python/MatrixInjector.py]: first two lines are for noPU workflows 5 lumis/job, last two lines are for PU workflows 1 lumi/job. I just take the same configs as for 2015 workflows.

**Comment by hengne** [comment]
There are the following errors:
1.) in unitTest, there is mainly the database connection error :
####### Case1: Connect to an invalid database.
CORAL/Services/XMLLookupService Error Could not open "dblookup.xml" for reading
# Expected Error: No physical Update connection for "invalid" is available. ( CORAL : "ReplicaCatalogue::replicasForConnection" from "CORAL/Services/ConnectionService" )
####### Case2: Connect to a valid database. Transaction not started.

2.) Matrix Test, there are two DAS errors.

All seem to be temporary connection errors to me. I will test it again.

**Comment by hengne** [comment]
please test

**Comment by hengne** [comment]
It is once more the DAS problem when reading MiniBias files for PU mixing:

```
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_1_X_2016-03-22-2300/external/slc6_amd64_gcc530/bin/das_client.py", line 448, in <module>
    main()
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_1_X_2016-03-22-2300/external/slc6_amd64_gcc530/bin/das_client.py", line 346, in main
    jsondict = get_data(host, query, idx, limit, debug, thr, ckey, cert)
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_1_X_2016-03-22-2300/external/slc6_amd64_gcc530/bin/das_client.py", line 253, in get_data
    fdesc = opener.open(req)
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/external/python/2.7.11-ikhhed/lib/python2.7/urllib2.py", line 437, in open
    response = meth(req, response)
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/external/python/2.7.11-ikhhed/lib/python2.7/urllib2.py", line 550, in http_response
    'http', request, response, code, msg, hdrs)
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/external/python/2.7.11-ikhhed/lib/python2.7/urllib2.py", line 475, in error
    return self._call_chain(*args)
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/external/python/2.7.11-ikhhed/lib/python2.7/urllib2.py", line 409, in _call_chain
    result = func(*args)
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/external/python/2.7.11-ikhhed/lib/python2.7/urllib2.py", line 558, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
urllib2.HTTPError: HTTP Error 503: Service Unavailable
```

**Comment by hengne** [comment]
please test

**Comment by hengne** [comment]
for the same das_client.py HTTPError i will wait until this issue has been solved to test again.

**Comment by boudoul** [comment]
Just for my own education, since this is the step3 which is failing because of query to the pile up dataset, , why do we have the pile up option in the step3 (reco step) ? I naively though the pile up option was needed only for the digi step ? (non urgent it's really for my own curiosity)

**Comment by hengne** [comment]
me too... i don't really understand...  the mixing should be done at the DIGI step. but would the RAW2DIGI in the reco step also makes such a das_client query?

**Comment by davidlange6** [comment]
the validation step effectively redoes all the mixing.. (apparently implemented as a second das query to get the files)

**Comment by boudoul** [comment]
ah it's the validation step , ok thanks!

**Comment by hengne** [comment]
please test

**Comment by hengne** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**